pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -202,26 +202,37 @@ func (m *manager) RemoveContainer(containerID string) error {
 }
 
 func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
-	klog.Infof("[topologymanager] Topology Admit Handler")
+	// Short circuit to unconditionally admit the pod if the policy is 'none'.
 	if m.policy.Name() == "none" {
-		klog.Infof("[topologymanager] Skipping calculate topology affinity as policy: none")
```

Comment:
I kind of like this log as it represents an exit path from the function and we log the other exit paths.

Reply:
+1